src/inverseRobot.js

```diff
@@ -8,6 +8,36 @@
 
 function inverseRobot(robot) {
   // write code here
+  const ONE_VALUE_IN_OBJECT = 1;
+  let countsObj = {};
+
+  const resObj = {};
+
+  if (Object.keys(robot).length === 0) {
+    return {};
+  }
+
+  for (const key in robot) {
+    if (countsObj[robot[key]] !== undefined) {
+      countsObj[robot[key]]++;
+    } else {
+      countsObj[robot[key]] = ONE_VALUE_IN_OBJECT;
+    }
+
+    countsObj = Object.assign(countsObj, countsObj[robot[key]]);
```

Comment:
The usage of 'Object.assign' here is unnecessary and incorrect. It should be removed